mits/site_sections/mits_applications.py

```diff
@@ -86,13 +90,13 @@ def delete_applicant(self, session, id):
     def picture(self, session, message='', image=None, **params):
         picture = session.mits_picture(params, applicant=True)
         if cherrypy.request.method == 'POST':
-            picture.filename = image.filename
```

Comment:
This change fixes a minor bug, which is that we'd give a 500 error page instead of returning a sensible error message if someone submitted the Upload Picture form without actually selecting a picture to upload.